### PR TITLE
Make colors important to override bootstrap or anything else

### PR DESCRIPTION
**Pull request by ajnassar**
- Make htk a little bit bigger from 200px to 350px
- Add variable for hover color of htk list item

This is related to this PR:
https://github.com/jontsai/maskil-www/pull/24
https://github.com/hacktoolkit/django-htk/pull/276

image
image

### CODE REVIEW

**Review by jontsai — APPROVED** [review]

File: _imports/colors.less [hunk 1]
```diff
@@ -56,6 +56,7 @@
 @color-button-green: rgb(76, 201, 71); /* this is a green */
 @color-button-maroon: rgb(202, 60, 60); /* this is a maroon */
 @color-button-light-blue: rgb(95, 198, 218); /* this is a light blue */
+@list-item-background-hover: #ffc;
```
Comment (jontsai) [inline _imports/colors.less]: this file should only be colors. I'd define this variable inside of `emulate_user.less` as it is more local to that only

**Review by jontsai — CHANGES_REQUESTED** [review]

File: _imports/colors.less (hunk 1 above)
Comment (ajnassar) [reply _imports/colors.less]: 👍

File: toolbar.less [hunk 2]
```diff
@@ -34,6 +22,7 @@
     }
 
     .htk-toolbar-panel {
+        @list-item-background-hover: #ffc;
```
Comment (jontsai) [inline toolbar.less:25]: @ajnassar we should call these variables

`@color-description: color-code;`

So in this case, `@color-list-item-background-hover: #ffc;`

**Review by jontsai — CHANGES_REQUESTED** [review]

**Review by jontsai — APPROVED** [review]
LFTM

**Review by jontsai — APPROVED** [review]
~LFTM.

Merging now, so we can make progress on it.

File: toolbar.less (hunk 2 above)
Comment (jontsai) [reply toolbar.less:25]: I took care of it in a follow up https://github.com/hacktoolkit/htk-css-hacktoolkit/commit/54360f2be9314e75d6f3b5741dec6b4490a9ac51

**Pull request merged**